clinica/pipelines/dwi_dti/dwi_dti_cli.py

```diff
@@ -4,46 +4,53 @@
 
 
 class DwiDtiCli(ce.CmdParser):
-
     def define_name(self):
         """Define the sub-command name to run this pipeline."""
-        self._name = 'dwi-dti'
+        self._name = "dwi-dti"
 
     def define_description(self):
         """Define a description of this pipeline."""
-        self._description = ('DTI-based processing of DWI datasets:\n'
-                             'http://clinica.run/doc/DWI_DTI')
+        self._description = (
+            "DTI-based processing of DWI datasets:\n"
+            "http://www.clinica.run/doc/Pipelines/DWI_DTI"
```

Comment:
Now that the documentation is stored in an alternative server, the url will be: https://aramislab.paris.inria.fr/clinica/docs/public/latest/Pipelines/DWI_DTI/
This change will be done also in another pipelines. Maybe we have to think about redirecting all the links automatically...

Reply:
I have wrote an Issue about it (#170) which is independent of this PR (the URL was wrong since the beginning 😅 ).